model_compression_toolkit/core/common/hessian/hessian_info_service.py

```diff
@@ -98,7 +98,7 @@ def compute(self, trace_hessian_request:TraceHessianRequest):
         Args:
             trace_hessian_request: Configuration for which to compute the approximation.
         """
-        Logger.info(f"Computing Hessian-trace approximation for a sample.")
+        Logger.debug(f"Computing Hessian-trace approximation for a node {trace_hessian_request.target_node}.")
```

Comment:
Why "debug"? what's the difference?

Reply:
To avoid populating the log with unused messages. The info messages will be when fetching approximations (the required number of approximations and the number of left to compute ones).